docs/source/cn/math_ops.py

```diff
@@ -3,41 +3,1196 @@
-        >>> y = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
-        >>> out = flow.add(x, y).numpy()
+        >>> x = flow.randn(2, 3, dtype=flow.float32)
+        >>> y = flow.randn(2, 3, dtype=flow.float32)
+        >>> out = flow.add(x, y)
         >>> out.shape
-        (2, 3)
+        oneflow.Size([2, 3])
 
         # 标量加法
         >>> x = 5
-        >>> y = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
-        >>> out = flow.add(x, y).numpy()
+        >>> y = flow.randn(2, 3, dtype=flow.float32)
+        >>> out = flow.add(x, y)
         >>> out.shape
-        (2, 3)
+        oneflow.Size([2, 3])
 
         # 广播加法
-        >>> x = flow.tensor(np.random.randn(1,1), dtype=flow.float32)
-        >>> y = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
-        >>> out = flow.add(x, y).numpy()
+        >>> x = flow.randn(1, 1, dtype=flow.float32)
+        >>> y = flow.randn(2, 3, dtype=flow.float32)
+        >>> out = flow.add(x, y)
+        >>> out.shape
+        oneflow.Size([2, 3])
+
+    """,
+)
+
+reset_docstr(
+    oneflow.sub,
+    r"""sub(input, other) -> Tensor
+    
+    计算 `input` 和 `other` 的差，支持 element-wise、标量和广播形式的减法。
+
+    公式为：
+
+    .. math::
+        out = input - other
+
+    参数：
+        - **input** (Tensor): 输入张量
+        - **other** (Tensor): 输入张量
+
+    返回值：
+        oneflow.Tensor: 结果张量
+    
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        
+        # element-wise 减法
+        >>> input = flow.randn(2, 3, dtype=flow.float32)
+        >>> other = flow.randn(2, 3, dtype=flow.float32)
+        >>> out = flow.sub(input,other)
+        >>> out.shape
+        oneflow.Size([2, 3])
+
+        # 标量减法
+        >>> input = 5
+        >>> other = flow.randn(2, 3, dtype=flow.float32)
+        >>> out = flow.sub(input,other)
+        >>> out.shape
+        oneflow.Size([2, 3])
+
+        # 广播减法
+        >>> input = flow.randn(1, 1, dtype=flow.float32)
+        >>> other = flow.randn(2, 3, dtype=flow.float32)
+        >>> out = flow.sub(input,other)
+        >>> out.shape
+        oneflow.Size([2, 3])
+
+    """,
+)
+
+reset_docstr(
+    oneflow.abs,
+    r"""abs(x) -> Tensor
+    
+    返回一个包含 `x` 中每个元素的绝对值的tensor:`y = |x|`。
+    
+    参数：
+        **x** (Tensor): 输入张量
+
+    返回值：
+        oneflow.Tensor: 结果张量
+
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+
+        >>> x = flow.tensor([-1, 2, -3, 4], dtype=flow.float32)
+        >>> flow.abs(x)
+        tensor([1., 2., 3., 4.], dtype=oneflow.float32)
+    
+    """,
+)
+
+reset_docstr(
+    oneflow.div,
+    r"""div(input, other) -> Tensor
+    
+    计算 `input` 除以 `other`，支持 element-wise、标量和广播形式的除法。
+
+    公式为：
+
+    .. math::
+        out = \frac{input}{other}
+    
+    参数：
+        - **input** (Union[int, float, oneflow.Tensor]): input.
+        - **other** (Union[int, float, oneflow.Tensor]): other.
+
+    返回值：
+        oneflow.Tensor: 结果张量
```

Comment:
```suggestion
    返回类型：
        oneflow.Tensor
```

我看了下，其他的pytorch中文版，以及oneflow英文原版，大部分没有专门给返回值做说明。
我们约定下，如果比较简单的单返回值，就给个返回类型就可以了吧。如果遇到复杂的多个返回值的情况，我们就再讨论。

这里个pr里上下有关的，都统一改一下。

注意，返回值和返回类型时不一样的。